src/modules/accommodations/entities/accommodation.entity.ts

```diff
@@ -37,10 +37,9 @@ export class Accommodation {
   images: Image[];
 
   @Column({
-    type: 'varchar',
-    length: 200,
+    type: 'json',
   })
-  location: string;
+  location: { latitude: number; longitude: number };
```

Comment:
I think It would be better if you stored them separately as new fields

Reply:
do you think we won't need a location later? like address, city, country, or optionally state (USA)?

Reply:
you should consult with our teacher or @farrooh about this